---a/net/minecraft/item/ItemBook.java
+++b/net/minecraft/item/ItemBook.java
@@ -1,17 +1,15 @@
 package net.minecraft.item;
 
-public class ItemBook extends Item
-{
+public class ItemBook extends Item {
+
     private static final String __OBFID = "CL_00001775";
 
-    public boolean isItemTool(ItemStack stack)
-    {
+    public boolean isItemTool(ItemStack stack) {
         return stack.stackSize == 1;
     }
 
-    public int getItemEnchantability()
-    {
+    public int getItemEnchantability() {
         return 1;
     }
-}
+}
